pages/_app.js

```diff
@@ -12,7 +12,7 @@ import ScrollButton from '../components/scrollToTop/ScrollButton';
 
 const Tablet = () => {
   const isTablet = useMediaQuery({ minWidth: 601, maxWidth: 1226 });
-  return isTablet;
+  return isTablet && false;
```

Comment:
This will return false for every page. Use an argument in `Tablet` function to determine page and then return false for `/developers` only

Reply:
@kaustubhai Ok sir,
Working on it now 🚀 🚀